examples/gym/gridworld_env.py

```diff
@@ -0,0 +1,87 @@
+import logging
+from typing import Tuple
+
+import numpy as np
+import gym
+
+
+ACTION_MAP = {0: (0, 1), 1: (1, 0), 2: (0, -1), 3: (-1, 0)}
+OBSTACLE_CHANNEL = 0
```

Comment:
is there an obstacle in this implementation? might be a cool way to demo masked actions

Reply:
hah, no obstacles yet, i just left a few incomplete hooks -- right now the step method doesn't take this channel into account.  but it would be easy to enable this, and yeah that would be a great way to demo masked actions.